### DISCUSSION TITLE
Stay in fullscreen mode when using jsPsych and JATOS

### DISCUSSION THREAD

**Discussion by steffejr** [jspsych/jsPsych#3391] [General]
Hello,
I am using JATOS to run a series of jsPsych experiments, one after the other. I would like to go to fullscreen mode at the start of the first experiment. Then, to stay in fullscreen mode until the last experiment finishes. Right now it will leave fullscreen mode at the end of each experiment and then ask the user at the start of the experiment. 
Is there any way to change this?
Thank you

**Comment by jodeleeuw** [comment]
Do you know if JATOS is what causes the experiment to exit fullscreen mode? I haven't used it much so I'm not familiar with how that works.

**Reply by steffejr** [reply]
Hello Josh,

I can find no mention of fullscreen in JATOS (documentation or code). In their forum, people have asked fullscreen questions when using OSWeb or OpenSeasame and they were directed to settings in OSWeb or OpenSeasame.
